packages/logger/tests/index.js

```diff
@@ -2,7 +2,7 @@
 
 const { Console } = require('console');
 const { Writable } = require('stream');
-const isString = require('lodash/isString');
+const isString = require('lodash.isstring');
 const moment = require('moment');
 const test = require('ava');
```

Comment:
I spent some time looking just now but couldn't figure out if Ava supports running your TS code directly rather than relying on the compiled JS code for tests

https://github.com/avajs/ava/blob/master/docs/recipes/typescript.md

Reply:
Actually, does the line below this still work..?

`const Logger = require('..');`

Just curious since we're no longer compiling `index.js` in the root but instead `dist/index.js`

Reply:
> I spent some time looking just now but couldn't figure out if Ava supports running your TS code directly rather than relying on the compiled JS code for tests

Ava runs against the compiled `.js` file that is generated. If we wanted better TS support in our tests, [Jest](https://jestjs.io/) seems to be what almost everyone recommends.

Reply:
> Actually, does the line below this still work..?
> 
> `const Logger = require('..');`
> 
> Just curious since we're no longer compiling `index.js` in the root but instead `dist/index.js`

Yeah, I was curious about that too. It does work, so `require()` must be respecting the `main` that we're setting in `package.json`.

Reply:
Interesting. If I open it up in VScode and click on the `require('..')` to open the file, it opens the type definition file. This is how most published packages behave, so that's fine

Reply:
However, while it is true that Ava runs the tests against the compiled JS files, that experience is not great. The compiled JS files aren't really human readable and putting breakpoints in there isn't really helpful because it's not the actual source code. It might be worth exploring this a little further

Reply:
It turns out that being able to set breakpoints in the TS code is as easy as enabling sourcemaps for the Typescript compilation. But I assume we only want this for local development. 

Is there any clever way to do `tsc --sourceMap` only for local development?

Reply:
Possibly if we use an env variable or an alternate package.json devel script chain (e.g. bootstrap-devel) that enables it.   

🤔

Reply:
We can build the source maps using `"sourceMap": true` in the `tsconfig.json` file, then update the `files` section of `package.json` to:

```json
"files": [
  "./dist/**/*.js",
  "./dist/**/*.d.ts",
],
```